shared/pages/Swap/DepositWindow/DepositWindow.js

```diff
@@ -92,25 +98,27 @@ export default class DepositWindow extends Component {
     }))
   }
 
-  updateRemainingBalance = () => {
+  updateRemainingBalance = async () => {
+    const { swap } = this.props
     const { sellAmount, balance } = this.state
-    const remainingBalance = BigNumber(sellAmount).minus(balance)
+    if (coinsWithDynamicFee.includes(swap.sellCurrency.toLowerCase())) {
+      const dynamicFee = await helpers[swap.sellCurrency.toLowerCase()].estimateFeeValue({ method: 'swap' })
+      const wantedBalanceWithoutFee = BigNumber(sellAmount).minus(balance)
+      const remainingBalance = (swap.sellCurrency === 'BTC' || swap.sellCurrency === 'ETH')
+        ? BigNumber(wantedBalanceWithoutFee).plus(dynamicFee)
+        : wantedBalanceWithoutFee
 
-    this.setState(() => ({
-      remainingBalance,
-    }))
+      this.setState(() => ({
+        remainingBalance,
```

Comment:
Лучше строку сделать, а то будет в таком виде передаваться:

![screenshot_20190211_222423](https://user-images.githubusercontent.com/43396375/52562950-ce4ba680-2e4b-11e9-8b16-b88e4cd45e52.png)

И если будет строка, то лучше убрать toFixed(), пусть всю длинну суммы показывает
`sellAmount.toFixed(6)
`
```suggestion
        remainingBalance: remainingBalance.toString(),
```